node/pkg/driver/node_utils.go

```diff
@@ -177,6 +179,23 @@ func (n NodeUtils) StageInfoFileIsExist(filePath string) bool {
 	}
 	return true
 }
+
+func (n NodeUtils) DevicesAreNvme(sysDevices []string) bool {
+	args := []string{"list"}
+	out, err := n.Executer.ExecuteWithTimeout(TimeOutMultipathdCmd, nvmeCmd, args)
```

Comment:
could we use a different timeout const here?
if you prefer, we can also have more general timeout consts, to use in specific timeout consts definition

Reply:
sure,
how do you suggest we call it? `TimeOutGeneralCmd`?

Reply:
if we only have mainly 1 "default" value, then sure

Reply:
done

Reply:
I didn't mean to replace existing specific timeout consts.
i meant we could have a more general const in addition.
e.g. (assuming you meant to have this value for both commands):
```
TimeOutGeneralCmd           = 10 * 1000
TimeOutMultipathdCmd        = TimeOutGeneralCmd
TimeOutNvmeCmd              = TimeOutGeneralCmd
```

Reply:
done